DQM/SiPixelPhase1Summary/src/SiPixelPhase1Summary.cc

```diff
@@ -272,6 +272,7 @@ void SiPixelPhase1Summary::fillSummaries(DQMStore::IBooker& iBooker, DQMStore::I
   //Fill the dead ROC summary
   std::vector<trendPlots> trendOrder = {layer1, layer2, layer3, layer4, ring1, ring2};
   std::vector<int> nRocsPerTrend = {1536, 3584, 5632, 8192, 4224, 6528};
+  std::vector<int> nDisabledRocs = {12, 128, 240, 320, 96, 120};
```

Comment:
These ROCs are disabled, they will never be recovered. The numbers are stable since June 2023. Hopefully the detector will not lose other modules in 2024 and 2025.